mods/cnc/rules/structures.yaml

```diff
@@ -803,7 +803,7 @@ SAM:
 	WithTurretedSpriteBody:
 	AutoSelectionSize:
 	Armament:
-		Weapon: SAMMissile
+		Weapon: Dragon
```

Comment:
This now leaves the `SAMMissile` counterintuitively only used by the advanced guard tower.  Please rename it to `TowerAAMissile` if this change is intentional.